src/robenv/templates/activate.template

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+. "${robenv_path}/opt/ros/${ros_distro}/setup.sh"
```

Comment:
shouldn't we source the one specific to the `$SHELL`? I could be wrong but I remember faintly that stuff like tab-autocompletion is dependant on `setup.bash` instead of `setup.sh`?

Reply:
The problem is the shebang. Thats needs to be done at file generation time and could be done in the future.